Misc/NEWS.d/3.9.0a1.rst

```diff
@@ -3372,7 +3372,7 @@ markup and any values in the message. Patch by Paul Ganssle
 .. section: Library
 
 Removed methods Element.getchildren(), Element.getiterator() and
-ElementTree.getiterator() and the xml.etree.cElementTree module.
```

Comment:
Don't remove news entries from published releases. For users, packagers and distributions it's important to know how the Python has changed between alpha releases.